packages/mix-models/src/rollback.ts

```diff
@@ -4,9 +4,10 @@ import set from 'lodash/set';
 
 import { Constructor } from './model';
 import { flattenKeys } from './flatten-keys';
+import { isArray } from 'lodash';
```

Comment:
Use direct lodash imports to make dependencies smaller.
```ts
import isArray from 'lodash/isArray';
```

But, this import is obsolete (use built-in `Array.isArray` instead) and should be places in the global imports sections (near another lodash imports).